heroic-core/src/main/java/com/spotify/heroic/common/CoreJavaxRestFramework.java

```diff
@@ -28,10 +28,13 @@
 import javax.ws.rs.container.CompletionCallback;
 import javax.ws.rs.container.ConnectionCallback;
 import javax.ws.rs.core.Response;
-import lombok.extern.slf4j.Slf4j;
+import org.slf4j.Logger;
 
-@Slf4j
 public final class CoreJavaxRestFramework implements JavaxRestFramework {
+
+    private static final Logger log =
+        org.slf4j.LoggerFactory.getLogger(CoreJavaxRestFramework.class);
```

Comment:
For all these log statements you should be able to qualify them as `LoggerFactory.getLogger()`that way its shorter and also fits on 1 line

Reply:
Yeah, those were all generated by the Intellij delombok function. I prefer the qualified version but not enough to manually go through and handcraft all those lines.